MEP_Engine/Query/EquipmentLengthByPart.cs

```diff
@@ -49,14 +35,21 @@ public static partial class Query
         [Description("Returns the length of the equipment based on the input of a list of parts")]
         [Input("airsidePartsByLength", "Collection of airside parts from a dataset that contains parts by length")]
         [Output("length", "The total length of the equipment based on the length of the parts")]
-        public static double EquipmentLengthByPart (this List<CustomObject> airsidePartsByLength)
+        public static double EquipmentLengthByPart (this List<IFlow> airsidePartsByLength)
```

Comment:
Looks fine to me, just needs some versioning I think :smile:

Reply:
Version = deleted